src/junit_reporter.js

```diff
@@ -306,6 +312,27 @@
             }
         };
 
+        self.formatSuiteData = function(suite) {
+            return {
+                disabled: suite._disabled || 0,
+                errors: 0,
```

Comment:
on second thought it probably makes sense to remove errors to reflect the structure of the inner testsuite tags.